### DISCUSSION TITLE
158

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#104] [General]
# 潮流周刊第158期 - 圣诞沙拉

封面图来源于周末提前过圣诞节去十八号酒馆吃了个圣诞餐，里面这个沙拉拍照效果很好，虾挺好吃，今天是圣诞节，祝你平安。

https://weekly.tw93.fun/posts/158-%E5%9C%A3%E8%AF%9E%E6%B2%99%E6%8B%89

**Comment by shawncai** [comment]
Waku：The minimal React framework这个链接地址好像放错了

**Reply by tw93** [reply]
哈哈 谢谢反馈 已经修复

**Comment by AlureLove** [comment]
发布日期是12/25哦，写成12/15啦~~

**Reply by tw93** [reply]
我的天.. 谢谢反馈，已经修复

**Comment by AlureLove** [comment]
信件已收到！

**Comment by zsj1029** [comment]
强烈建议增加dark模式
